src/helpers.js

```diff
@@ -89,6 +89,32 @@ module.exports = ({ types: t }) => {
     return scope.hasOwnBinding(name) ? scope.generateUidIdentifier(name) : t.identifier(name);
   };
 
+  const isFunctionExpression = factory => {
+    return t.isFunctionExpression(factory) || t.isArrowFunctionExpression(factory);
+  };
+
+  const createFunctionExpression = (factory, requireExpressions) => {
```

Comment:
Since this method takes in requireExpressions, let's rename it to something like `createFactoryReplacementExpression`, since `createFunctionExpression` is a bit too generic.